activiti-services/activiti-services-connectors/src/main/java/org/activiti/services/connectors/model/AsyncContext.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.model;
+
+public class AsyncContext {
+
+    private String processInstanceId;
```

Comment:
@erdemedeiros as I mention before we might need here a correlationId. This structure is the one that we need to store, are you keeping these instances somewhere? Not about the name yet.

Reply:
@Salaboy I'm not sure that we need to store all this information. IMO processInstanceId and TaskId are not necessary.